pwndbg/commands/context.py

```diff
@@ -834,6 +834,56 @@ def context_args(with_banner=True, target=sys.stdout, width=None):
 last_signal: List[str] = []
 
 
+def get_thread_status(thread):
+    if thread.is_running():
+        return pwndbg.color.light_green("running")
```

Comment:
nitpick: if those colors are constant/not-changeable, lets compute/move those colored-strings once by moving them to the global scope.